configure.ac

```diff
@@ -93,11 +93,16 @@ test -z "${tmpfilespath}" && tmpfilespath=/usr/lib/tmpfiles.d
 AC_SUBST(SYSTEMD_TMPFILESDIR, [${tmpfilespath}])
 
 AC_ARG_WITH([systemdsysctldir], AS_HELP_STRING([--with-systemdsysctldir=DIR],
-	    [path to systemd sysctl dir @<:@default=/usr/lib/sysctl.d@:>@]), [sysctlpath=${withval}],
-	    [sysctlpath="$($PKG_CONFIG --variable=sysctldir systemd)"])
+            [path to systemd sysctl dir @<:@default=/usr/lib/sysctl.d@:>@]), [sysctlpath=${withval}],
+            [sysctlpath="$($PKG_CONFIG --variable=sysctldir systemd)"])
```

Comment:
whitespace changes... Looks like the rest of the file uses tabs+spaces, so I would switch it back for consistency.

Reply:
lovely... I'll fix